docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Regression/LightGbmWithOptionsAdvanced.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using Microsoft.ML.Trainers.LightGbm;
+
+namespace Samples.Dynamic.Trainers.Regression
+{
+    class LightGbmWithOptionsAdvanced
```

Comment:
>LightGbmWithOptionsAdvanced [](start = 10, length = 27)

Is there a tt for this?

Reply:
No for the same reason to LightGbmAdvanced.cs

---
In reply to: [275456961](https://github.com/dotnet/machinelearning/pull/3319#discussion_r275456961) [](ancestors = 275456961)